packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/relaunch/relaunch.controller.js

```diff
@@ -0,0 +1,64 @@
+export default class TelephonyPortabilitiesRelaunchCtrl {
+  /* @ngInject */
+  constructor($translate, TelephonyPortabilitiesService) {
+    this.$translate = $translate;
+    this.TelephonyPortabilitiesService = TelephonyPortabilitiesService;
+    this.isRelaunching = false;
+  }
+
+  $onInit() {
+    this.model = {
+      mandatoryParameters: Object.fromEntries(
+        Object.values(this.changeRequired.mandatoryParameters).map((value) => {
+          return [value, null];
+        }),
+      ),
+      optionalParameters: Object.fromEntries(
+        Object.values(this.changeRequired.optionalParameters).map((value) => {
+          return [value, null];
+        }),
+      ),
```

Comment:
**suggestion:** Array.prototype.reduce can be better solution
```suggestion
      optionalParameters: this.changeRequired.optionalParameters.reduce((acc, value) => {
          acc[value] = null;
          return acc;
        }),
```

Reply:
It's applied.